packages/atlas/src/components/_navigation/SidenavBase/SidenavBase.styles.ts

```diff
@@ -189,30 +129,29 @@ export const LegalLinksWrapper = styled.span`
 
 export const LegalLink = styled(Link)`
   text-decoration: none;
-  font: ${cVar('typographyDesktopT200Strong')};
-  letter-spacing: ${cVar('typographyDesktopT200StrongLetterSpacing')};
-  text-transform: ${cVar('typographyDesktopT200StrongTextTransform')};
+  color: ${cVar('colorTextMuted')} !important;
```

Comment:
Why is important needed here?

Reply:
That's not true, you passed `color: inherit` to the `Text` component. You can remove this `!important` and the color stays correct